stable/database/templates/_sidecar.tpl

```diff
@@ -5,6 +5,9 @@
   image: {{ template "nuocollector.image" . }}
   imagePullPolicy: {{ .Values.nuocollector.image.pullPolicy }}
   tty: true
+  resources:
+{{ toYaml .Values.nuocollector.resources | trim | indent 4 }}
```

Comment:
I think you should be able to intend these normally if you use `{{- ... }}`.

Reply:
I will try with `{{- toYaml ... | nindent 4 }}` so that we keep the leading new line.